pkg/apis/extensions/types.go

```diff
@@ -676,6 +676,12 @@ type PodSecurityPolicySpec struct {
 	// If set to false the container may run with a read only root file system if it wishes but it
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty"`
+	// SeccompProfiles lists the allowed profiles that may be set for the pod or
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
+	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
+	// used to generate a value for a pod the first non-wildcard profile will be used as
```

Comment:
Ok, here's my summary:

1.  With seccomp currently you can submit any annotation you want and it's allowed and honored by the kubelet. 
1.  If you don't provide a profile you're getting unconfined by default from the kubelet

For item 1 - I liken this to `privileged`.  If you enable PSP you don't get that behavior anymore by default unless your policies are very lenient.  The cluster defaults to a more secure state.

For item 2 - this is the same as disallowing all profiles (IMO).  If you disallow all profiles then you are requesting the cluster to act in a sane way on your behalf.

https://github.com/kubernetes/kubernetes/pull/28300#discussion_r69901020 is very interesting since we already support granular capability allowances, defaulting, and restriction.  Although the backwards compat will still require us to be able to restrict full profile references.